superset/assets/javascripts/explorev2/components/QueryAndSaveBtns.jsx

```diff
@@ -9,7 +9,7 @@ const propTypes = {
   onSave: PropTypes.func,
   onStop: PropTypes.func,
   loading: PropTypes.bool,
-  errorMessage: PropTypes.string,
+  errorMessage: PropTypes.node,
```

Comment:
could use `!!` to convert the truthy/falsey value to a boolean.

`disabled={!!errorMessage}`